src/main/java/io/cdap/plugin/gcp/spanner/sink/SpannerOutputFormat.java

```diff
@@ -50,11 +50,11 @@
    * @param schema        schema for spanner table
    */
   public static void configure(Configuration configuration, SpannerSinkConfig config, Schema schema) {
-    String projectId = config.getProject();
+    String projectId = config.connection.getProject();
```

Comment:
Where does this method get called? Since it is named as `configure`, if it is called in `configurePipeline` method, this change will cause NPE

Reply:
It is called in the `prepareRun`. https://github.com/data-integrations/google-cloud/blob/develop/src/main/java/io/cdap/plugin/gcp/spanner/sink/SpannerSink.java#L140